### PR TITLE
Match mists unauthorised output

### PR DESCRIPTION
**Pull request by mjh1**
So that the UI can continue to check for this and display the correct messages to the user

### CODE REVIEW

File: middleware/gating.go [hunk 1]
```diff
@@ -33,20 +33,29 @@ func (h *GatingHandler) GatingCheck(next httprouter.Handle) httprouter.Handle {
 		playbackAccessControlAllowed, err := h.AccessControl.IsAuthorized(playbackID, req.URL)
 		if err != nil {
 			log.LogError(requestID, "unable to get playback access control info", err, "playbackID", playbackID, playback.KeyParam, key)
-			deny(w)
+			catErrs.WriteHTTPInternalServerError(w, "unable to get playback access control info", nil)
```
Comment (victorges) [inline middleware/gating.go:36]: **nit:** WDYT
```suggestion
			catErrs.WriteHTTPInternalServerError(w, "unable to authorize playback request", nil)
```

**Review by victorges — APPROVED** [review]
LGTM!

File: middleware/gating.go (hunk 1 above)
Comment (mjh1) [reply middleware/gating.go]: looks good 👍

File: middleware/gating.go [hunk 2]
```diff
@@ -33,20 +33,29 @@ func (h *GatingHandler) GatingCheck(next httprouter.Handle) httprouter.Handle {
...
 
 		next(w, req, params)
 	}
 }
 
-func deny(w http.ResponseWriter) {
-	catErrs.WriteHTTPUnauthorized(w, "denied", errors.New("access control failed"))
+func deny(requestFile string, w http.ResponseWriter) {
+	if !playback.IsManifest(requestFile) {
+		catErrs.WriteHTTPUnauthorized(w, "unauthorised", errors.New("access control denied"))
```
[the remark this answers is not in the document]
Comment (mjh1) [reply middleware/gating.go:52]: good spot thanks

**Pull request merged**